.gitignore

```diff
@@ -30,6 +30,7 @@ VERSION
 /test/results
 coverage.txt
 *.log
+test/integration/test-fixtures/**/go.sum
```

Comment:
I keep running into goland generating go.sums for these and we don't want them in the codebase